### PR TITLE
feat: Allow generating code without a prefix

### PR DESCRIPTION
**Pull request by SferaDev**

### CODE REVIEW

File: plugins/typescript/src/generators/generateReactQueryComponents.ts [hunk 1]
```diff
@@ -65,7 +65,12 @@ export const generateReactQueryComponents = async (
       .join("\n");
 
   const filenamePrefix =
-    c.snake(config.filenamePrefix || context.openAPIDocument.info.title) + "-";
+    c.snake(
+      isNull(config.filenamePrefix)
+        ? ""
+        : config.filenamePrefix || context.openAPIDocument.info.title
```
Comment (fabien0102) [inline plugins/typescript/src/generators/generateReactQueryComponents.ts:71]: Now that you point at this issue, I think we should use `??` instead `||` so we can setup `filePrefix` to `""` or `null`

File: plugins/typescript/src/generators/generateReactQueryComponents.test.ts [hunk 2]
```diff
@@ -76,6 +77,7 @@ describe("generateReactQueryComponents", () => {
         openAPIDocument,
         writeFile,
         existsFile: () => true,
+        readFile: async () => "",
```
Comment (fabien0102) [inline plugins/typescript/src/generators/generateReactQueryComponents.test.ts:80]: Sorry for that, this fix on my other PR (#49) also normally

File: plugins/typescript/src/generators/generateReactQueryComponents.ts (hunk 1 above)
Comment (SferaDev) [reply plugins/typescript/src/generators/generateReactQueryComponents.ts:71]: Oh yeah, although it wouldn't be ``null`` just empty string

Comment (SferaDev) [reply plugins/typescript/src/generators/generateReactQueryComponents.ts]: PR updated, makes more sense now

**Review by fabien0102 — APPROVED** [review]

**Pull request merged**